android/src/com/frostwire/android/StoragePicker.java

```diff
@@ -40,7 +40,7 @@
 
     public static final String ACTION_OPEN_DOCUMENT_TREE = "android.intent.action.OPEN_DOCUMENT_TREE";
 
-    public static final int SELECT_FOLDER_REQUEST_CODE = 1267123;
+    public static final int SELECT_FOLDER_REQUEST_CODE = 126;
```

Comment:
from grzesiek "seems like it is an internal android thing that when we switched to compat version the requestCode for starting activity has to be a 16bit unsigned integer"